db/methods/email_domain_blacklist/update.js

```diff
@@ -0,0 +1,11 @@
+const { updateDocument } = require('../../dynamo');
+const EmailDomainBlacklist = require('../../schemas/email_domain_blacklist');
+
+function updateAccount(domainName, blacklistEntry) {
+    return updateDocument(EmailDomainBlacklist, {
+        domainName,
+        ...blacklistEntry,
+    });
+}
```

Comment:
Not as far as I know, no. Definitely worth looking into to maintain data integrity going forward, but probably not as big of an issue as it was for us in our past since the service layer was refactored from something that accepted only a strict subset of parameters.